data-serving/data-service/api/dataserver.api.yaml

```diff
@@ -0,0 +1,54 @@
+openapi: 3.0.3
```

Comment:
Since this is already in the data-service dir, we should use the recommended root spec file name of `openapi.yaml`. Or is your plan that other APIs might produce such a specification, and we'd organize a single epid root spec?

Reply:
nah probably just this one! renamed